lighthouse-core/test/config/config-test.js

```diff
@@ -246,20 +246,20 @@ describe('Config', () => {
           }
         },
       ],
-    }), /no failureDescription and should/);
+    }), /no failureTitle and should/);
 
     assert.throws(_ => new Config({
-      audits: [basePath + '/missing-help-text'],
-    }), /meta.helpText property/);
+      audits: [basePath + '/missing-description'],
+    }), /meta.description property/);
 
     assert.throws(_ => new Config({
       audits: [
         class EmptyStringHelpTextAudit extends Audit {
```

Comment:
> hm @paulirish where? vscode isn't finding any more non-string ones :/

yeah i was just looking for any of them. relax that case sensitivity ;)

helptext -> description

Reply:
fixed